Projects/My_website/myWebsite/account/views.py

```diff
@@ -1,6 +1,37 @@
-from django.shortcuts import render
-
+from django.shortcuts import render,redirect
+from django.contrib.auth.models import User,auth
+from django.contrib import messages
 # Create your views here.
 
 def login(request):
-    return render(request,'account/login.html')
\ No newline at end of file
+    if request.method=='POST':
+        username=request.POST["username"]
+        password = request.POST["password"]
+        user = auth.authenticate(username=username,password=password)
+        if user is not None:
```

Comment:
@codal-uchoudhary , instead of writing _is not None_, you can simply write _if user:_